Description: Fix the space glyph in Samyak Tamil
 This is a patch to fix the issue raised by the bug #687176
 .
Bug-Debian: http://bugs.debian.org/687176
Author: Kartik Mistry <kartik@debian.org>
Last-Update: 2012-09-15
 
--- a/tamil/Samyak_Tamil.sfd
+++ b/tamil/Samyak_Tamil.sfd
@@ -69,14 +69,12 @@ AntiAlias: 1
 FitToEm: 1
 WinInfo: 0 39 15
 BeginChars: 65602 117
-StartChar: .notdef
-Encoding: 32 32 0
-Width: 443
+StartChar: space
+Encoding: 32 32 3
+Width: 1200
 GlyphClass: 2
 Flags: W
-TtfInstrs: 46
-YlOhX4L,1p!:;PH"pNdEZ3(..m4n[H!rsu:Z3:@2m4tsP"p+WaZ2k",m4nYA
-EndTtf
+LayerCount: 2
 EndChar
 StartChar: Pulli
 Encoding: 2946 2946 1
